core/background/objects/song.js

```diff
@@ -50,15 +50,14 @@ define([
 		/**
 		 * Various optional data
 		 */
-		var connector = _.find(connectors, function(connector) {
-			return _.find(connector.matches, function(patternStr) {
-				var pattern = RoutePattern.fromString(patternStr);
```

Comment:
It was searching for matches between some URL match pattern and some URL.
The code in `inject.js` does it better.